etc/profile-m-z/parsecd.profile

```diff
@@ -0,0 +1,22 @@
+include parsecd.local
```

Comment:
Please add the headers we use in all .profile files. There's a template for that [here](https://github.com/netblue30/firejail/blob/master/etc/templates/profile.template).